airbyte-integrations/bases/base-java-s3/src/main/java/io/airbyte/integrations/destination/s3/csv/StagingDatabaseCsvSheetGenerator.java

```diff
@@ -33,7 +33,7 @@ public class StagingDatabaseCsvSheetGenerator implements CsvSheetGenerator {
   private final List<String> header;
 
   public StagingDatabaseCsvSheetGenerator() {
-    this(false);
+    this(true);
```

Comment:
why are we switching the default behavior? this class is also used in destination-redshift, which isn't yet using v2 tables

Reply:
Switch it back and use the non default constructor in BigQuery